cli/client/http.go

```diff
@@ -5,68 +5,85 @@ import (
 	"crypto/tls"
 	"crypto/x509"
 	"fmt"
-	"github.com/mesosphere/dcos-commons/cli/config"
 	"io/ioutil"
 	"log"
 	"net/http"
 	"net/url"
 	"path"
 	"strings"
+
+	"github.com/mesosphere/dcos-commons/cli/config"
 )
 
-func HTTPGet(urlPath string) *http.Response {
-	return CheckHTTPResponse(HTTPQuery(CreateHTTPRequest("GET", urlPath)))
+func HTTPServiceGet(urlPath string) *http.Response {
```

Comment:
I'd like to be explicit - it may be the case in the future that we add support for querying other services on DC/OS directly (e.g. Marathon). What do you think?